runtime/vm/JFRChunkWriter.hpp

```diff
@@ -188,6 +189,7 @@ class VM_JFRChunkWriter {
 	static constexpr int THREAD_CPU_LOAD_EVENT_SIZE = (2 * sizeof(float)) + (4 * sizeof(I_64));
 	static constexpr int INITIAL_ENVIRONMENT_VARIABLE_EVENT_SIZE = 6000;
 	static constexpr int CLASS_LOADING_STATISTICS_EVENT_SIZE = 5 * sizeof(I_64);
+	static constexpr int CLASS_LOADER_STATISTICS_EVENT_SIZE = (3 * LEB128_32_SIZE) + (9 * LEB128_64_SIZE);
```

Comment:
`ClassLoaderStatisticsEntry` has 2 32-bit fields and 8 64-bit fields; the size accounts for a third 32-bit field: What is the ninth 64-bit field?